packages/commonwealth/client/scripts/views/pages/new_proposal/sputnik_proposal_form.tsx

```diff
@@ -81,46 +124,7 @@ export const SputnikProposalForm = () => {
           }}
         />
       )}
-      <CWButton
-        label="Send transaction"
-        onClick={(e) => {
-          e.preventDefault();
-
-          let propArgs: NearSputnikProposalKind;
-
-          if (sputnikProposalType === 'addMember') {
-            propArgs = {
-              AddMemberToRole: { role: 'council', member_id: member },
-            };
-          } else if (sputnikProposalType === 'removeMember') {
-            propArgs = {
-              RemoveMemberFromRole: { role: 'council', member_id: member },
-            };
-          } else if (sputnikProposalType === 'payout') {
-            let amount: string;
-            // treat NEAR as in dollars but tokens as whole #s
-            if (!tokenId) {
-              amount = app.chain.chain
-                .coins(+payoutAmount, true)
-                .asBN.toString();
-            } else {
-              amount = `${+payoutAmount}`;
-            }
-
-            propArgs = {
-              Transfer: { receiver_id: member, token_id: tokenId, amount },
-            };
-          } else if (sputnikProposalType === 'vote') {
-            propArgs = 'Vote';
-          } else {
-            throw new Error('unsupported sputnik proposal type');
-          }
-
-          (app.chain as NearSputnik).dao
-            .proposeTx(description, propArgs)
-            .catch((err) => notifyError(err.message));
-        }}
-      />
+      <CWButton label="Send transaction" onClick={handleSendTransaction} />
```

Comment:
thanks for moving all of these inline functions into the handlers! This way we declutter jsx/visual part of the code